src/command_modules/azure-cli-cosmosdb/azure/cli/command_modules/cosmosdb/tests/latest/test_cosmosdb_commands.py

```diff
@@ -158,3 +158,16 @@ def test_locations_database_accounts(self, resource_group):
         assert account2['writeLocations'][0]['locationName'] == "West US"
         assert account2['readLocations'][0]['locationName'] == "East US" or account2['readLocations'][1]['locationName'] == "East US"
         assert account2['readLocations'][0]['failoverPriority'] == 1 or account2['readLocations'][1]['failoverPriority'] == 1
+
+    @ResourceGroupPreparer(name_prefix='cli_test_cosmosdb_account')
+    def test_enable_multiple_write_locations(self, resource_group):
```

Comment:
This test is failing on the CI.

Reply:
The azure-mgmt-cosmosdb library had not yet been updated to include that parameter. The test passes now.